components/eamxx/src/control/atmosphere_driver.cpp

```diff
@@ -1620,9 +1620,11 @@ void AtmosphereDriver::run (const int dt) {
   EKAT_REQUIRE_MSG (dt>0, "Error! Input time step must be positive.\n");
 
   // Print current timestamp information
+  auto end_of_step = m_current_ts + dt;
   m_atm_logger->log(ekat::logger::LogLevel::info,
-    "Atmosphere step = " + std::to_string(m_current_ts.get_num_steps()) + "\n" +
```

Comment:
I think seeing the counter printed reflecting the steps completed _before_ the current one was a bit confusing. When running the 124th step, I think ppl would like to see `Atmosphere step = 124`...